raiden/transfer/channel.py

```diff
@@ -874,6 +882,14 @@ def get_next_nonce(end_state: NettingChannelEndState) -> typing.Nonce:
     return 1
 
 
+def _merkletree_width(merkletree) -> int:
```

Comment:
`merkletree` is of type `MerkleTreeState`

Reply:
Thanks, added.